strtoint/strtoint3.go

```diff
@@ -0,0 +1,10 @@
+package strtoint
+
+import (
+	"fmt"
+)
+
+func MyStrToInt3(s string) (i int, err error) {
+	_, err = fmt.Sscanf(s, "%d", &i)
```

Comment:
Haha! I thought none will find this solution. Sweet!

Reply:
Yeah, by other languages i know that it's not a good solution to have it in repo, but idk as it is accepted in go community :) https://github.com/VSKut/go-temp/pull/9/commits/cf8ef5973e777b40fef68627d4751deff0aa86ae